ray/operator/base/params.env

```diff
@@ -1 +1,2 @@
 namespace=opendatahub
+odh-kuberay-operator-controller-image=quay.io/opendatahub/kuberay-operator:v0.5.0
```

Comment:
Should we use `quay.io/opendatahub/kuberay-operator:v0.6.0`

Reply:
I think we should upgrade as part of a PR to DW first, and then copy the manifests in here.